plugins/chat/assets/javascripts/discourse/models/chat-channel.js

```diff
@@ -116,6 +116,7 @@ export default class ChatChannel {
     }
 
     this.tracking = new ChatTrackingState(getOwner(this));
+    this.lastMessage = ChatMessage.create(this, args.last_message || {});
```

Comment:
should we create an instance of ChatMesage is we dont have `args.last_message` ?

Reply:
In https://github.com/discourse/discourse/pull/22488/commits/adf089941abfb0da8c50309c4debe293d00f3044 I decided to only have a last message if it exists. A channel can be created with no messages.